masterlist.yaml

```diff
@@ -19150,11 +19150,19 @@ plugins:
   # Main Mod #
   - name: 'UltimateCollege.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/44797' ]
+    after:
+      - 'mihailspectres.esp'
+      - 'mihailbeholder.esp'
+      - 'ImmersiveIntegrationSE15.esp'
     inc:
       - 'CollegeModifications.esp'
       - 'MagicalCollegeofWinterhold.esp'
       - 'OCW_LightingPatch_Brighter.esp'
       - 'OCW_LightingPatch_Darker.esp'
+      - name: 'mihailspectres.esp'
+        condition: 'active("mihailspectres.esp") and checksum("mihailspectres.esp", 0x95F0F68D)'
```

Comment:
You don't need the prefix of the CRC when using `checksum()`.
```
        condition: 'active("mihailspectres.esp") and checksum("mihailspectres.esp", 95F0F68D)'
```

Reply:
Sorry for that